packages/wallets/src/components/WalletsAddMoreCardBanner/WalletsAddMoreCardBanner.tsx

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
+import type { useAvailableWallets } from '@deriv/api';
+import CheckIcon from '../../public/images/check.svg';
+import PlusIcon from '../../public/images/plus.svg';
+import WalletCurrencyIcon from '../WalletCurrencyIcon';
 
-const WalletsAddMoreCardBanner = ({
-    is_added,
-    landing_company_name,
-}: {
-    is_added: boolean;
-    landing_company_name: string;
-}) => {
+type TWalletsAddMoreCardBannerProps = NonNullable<ReturnType<typeof useAvailableWallets>['data']>[0];
+
+const WalletsAddMoreCardBanner = ({ currency, is_added, landing_company_name }: TWalletsAddMoreCardBannerProps) => {
     return (
         <div className='wallets-add-more__banner'>
             <div className='wallets-add-more__banner-header'>
-                <span className='wallets-add-more__banner-logo'>LOGO</span>
-                <span className='wallets-add-more__banner-landing-company'>{landing_company_name.toUpperCase()}</span>
+                <span className='wallets-add-more__banner-logo'>
+                    <WalletCurrencyIcon currency={currency ? currency.toLowerCase() : ''} />
+                </span>
+                <span className='wallets-add-more__banner-landing-company'>
+                    {landing_company_name ? landing_company_name.toUpperCase() : ''}
+                </span>
             </div>
             <button
                 className={`wallets-add-more__banner-button ${
                     is_added ? 'wallets-add-more__banner-button--is-added' : ''
                 }`}
             >
-                {is_added ? '✅ Added' : '🏥 Add'}
+                {is_added ? (
+                    <CheckIcon className='wallets-add-more__banner-button-icon' />
+                ) : (
+                    <PlusIcon className='wallets-add-more__banner-button-icon' />
+                )}
```

Comment:
Styled buttons and imported new SVG icons from Figma design: 
<img width="982" alt="Screenshot 2023-09-18 at 3 35 33 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/caf12a31-93af-4461-bab9-9005ddbc8b88">